harness/determined/common/experimental/determined.py

```diff
@@ -155,25 +155,23 @@ def create_model(
 
         return model.Model._from_json(r.json().get("model"), self._session)
 
-    def get_model(self, name: str) -> model.Model:
+    def get_model(self, identifier: Union[str, int]) -> model.Model:
         """
         Get the :class:`~determined.experimental.Model` from the model registry
-        with the provided name. If no model with that id is found in the registry,
+        with the provided name or ID. If no corresponding model is found in the registry,
         an exception is raised.
         """
-        r = self._session.get("/api/v1/models/{}".format(name))
-        return model.Model._from_json(r.json().get("model"), self._session)
+        r = self._session.get("/api/v1/models/{}".format(identifier)).json()
```

Comment:
can you update this to use the newer f-string style (`.get(f"/api/v1/models/{identifier}"`)?

The reason this code was written with `.format()` was because it was written when the python cli still had to support python 3.5, but that's no longer true.